rpcs3/rpcs3qt/tooltips.h

```diff
@@ -231,6 +231,7 @@ class Tooltips : public QObject
 		const QString buzz              = tr("Buzz! support.\nSelect 1 or 2 controllers if the game requires Buzz! controllers and you don't have real controllers.\nSelect Null if the game has support for DualShock or if you have real Buzz! controllers.");
 		const QString turntable         = tr("DJ Hero Turntable controller support.\nSelect 1 or 2 controllers if the game requires DJ Hero Turntable controllers and you don't have real turntable controllers.\nSelect Null if the game has support for DualShock or if you have real turntable controllers.\nA real turntable controller can be used at the same time as an emulated turntable controller.");
 		const QString ghltar            = tr("Guitar Hero Live (GHL) Guitar controller support.\nSelect 1 or 2 controllers if the game requires GHL Guitar controllers and you don't have real guitar controllers.\nSelect Null if the game has support for DualShock or if you have real guitar controllers.\nA real guitar controller can be used at the same time as an emulated guitar controller.");
+		const QString gametablet        = tr("uDraw GameTablet emulated controller support.\nDisable to use a physical uDraw GameTablet device.\n• uDraw Studio: Instant Artist  requires to configure Mouse Handler=Basic, Pad1=Connected and Pad2=Null.\n• Marvel Super Hero Squad: Comic Combat  requires to configure Mouse Handler=Basic, Pad1=Null and Pad2=Connected.");
```

Comment:
Why are there 2 spaces after each of the game names?
Also, I don't think it makes sense to mention specific games. There are more than 2 games that use udraw, and this is something that belongs to the wiki.
I had something more general in mind. Like a hint that you might need one of the two settings.
```suggestion
		const QString gametablet        = tr("uDraw GameTablet emulated controller support.\nDisable to use a physical uDraw GameTablet device.\n• uDraw Studio: Instant Artist  requires to configure Mouse Handler=Basic, Pad1=Connected and Pad2=Null.\n• Marvel Super Hero Squad: Comic Combat  requires to configure Mouse Handler=Basic, Pad1=Null and Pad2=Connected.");
```

Reply:
I added 2 spaces to improve the legibility, Without rich text this was the method I found. 
I found only 3 PS3 games with uDraw support (Wii have more), and the third game works with any of these 2 settings.